### ISSUE TITLE
Removing "com.oneplus.simcontacts" can crash "com.android.dialer"

### ISSUE THREAD

**Issue by KingEik** [0x192/Universal-Android-Debloater#304]
**Your phone**: OnePlus 5T

**Describe the issue**
Removing "com.oneplus.simcontacts" makes the dialer app ("Phone" ; "com.android.dialer") crash if it tries to display the call history.
Exception message (from Logcat): "Failed to find provider com.oneplus.simcontacts for user 0; expected to find a valid ContentProvider for this authority"

**You have a solution?**
I would propose moving "com.oneplus.simcontacts" from "Recommended" to "Advanced" or "Expert" and updating the description to reflect the possibility of this issue occuring.

**Comment by AnonymousWP** [comment]
> **Your phone**: OnePlus 5T
> 
> **Describe the issue** Removing "com.oneplus.simcontacts" makes the dialer app ("Phone" ; "com.android.dialer") crash if it tries to display the call history. Exception message (from Logcat): "Failed to find provider com.oneplus.simcontacts for user 0; expected to find a valid ContentProvider for this authority"
> 
> **You have a solution?** I would propose moving "com.oneplus.simcontacts" from "Recommended" to "Advanced" or "Expert" and updating the description to reflect the possibility of this issue occuring.

Interesting. Do you perhaps know whether this also happens on newer versions of OxygenOS? Because your 5T is probably running OxygenOS 10.0.1, which is still based on Android 10 (and the latest update the phone received).

**Comment by KingEik** [comment]
Sorry, but I don't have any devices to check this with.
Small addition: My 5T is actually still running OxygenOS 9.0.4 but I don't know if this makes a difference for this issue.

**Comment by AnonymousWP** [comment]
> Sorry, but I don't have any devices to check this with. Small addition: My 5T is actually still running OxygenOS 9.0.4 but I don't know if this makes a difference for this issue.

Could you try upgrading to 10.0.1 and see if the issue still persists? Then we can know whether this is a bug or not. Also, staying on 9.0.4 with a very old security patch increases the attack vector. Do mind that updating to 10.0.1 likely introduces new APKs/bloat and also removes some.

**Comment by KingEik** [comment]
I can do that but it will take me some time (rooted phone, therefore some effort required to update ^^).

**Comment by AnonymousWP** [comment]
> I can do that but it will take me some time (rooted phone, therefore some effort required to update ^^).

Yeah, that's true. Good luck and let me know once you know more. 🙂

**Comment by justaguydoinghisstuff** [comment]
I have tried both uninstalling and "clear and disable..." and i see no issues
OP6T OxygenOS 10.3.12
